api_tests/actions/views/test_schema_response_action_detail.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+
+from osf_tests.factories import (
+    AuthUserFactory,
+    RegistrationFactory,
+    SchemaResponseFactory,
+    SchemaResponseActionFactory
+)
+
+
+@pytest.mark.django_db
+@pytest.mark.enable_quickfiles_creation
+class TestSchemaResponseActionDetail:
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def registration(self):
+        return RegistrationFactory()
+
+    @pytest.fixture()
+    def schema_response(self, registration):
+        return SchemaResponseFactory(
+            registration=registration,
+            initiator=registration.creator,
+        )
+
+    @pytest.fixture()
+    def schema_response_action(self, schema_response):
+        return SchemaResponseActionFactory(
+            target=schema_response,
+        )
+
+    @pytest.fixture()
+    def url(self, schema_response, schema_response_action):
+        return f'/v2/schema_responses/{schema_response._id}/actions/{schema_response_action._id}/'
+
+    def test_schema_response_action_detail(self, app, schema_response_action, user, url):
+        resp = app.get(url, auth=user.auth)
+        assert resp.status_code == 200
+        data = resp.json['data']
+        assert schema_response_action._id == data['id']
+
+    @pytest.mark.parametrize(
+        'permission,expected_response',
+        [
+            (None, 200, ),
```

Comment:
I disagree with a non-contributor probably shouldn't be able to see actions on a non-approved response, but this is technically what's in the doc, so is probably scope creep to suggest otherwise...